thorlcr/master/thgraphmanager.cpp

```diff
@@ -189,7 +189,7 @@ class CIdleShutdown : public CSimpleInterface, implements IThreaded
     virtual void main()
     {
         if (!sem.wait(timeout)) // feeling neglected, restarting..
-            abortThor(MakeThorException(TE_IdleRestart, "Thor has been idle for %d minutes, restarting", timeout/60000));
+            abortThor(MakeThorException(TE_IdleRestart, "Thor has been idle for %d minutes, restarting", timeout/60000), TEC_Idle, false);
```

Comment:
NB: There's an incidental change here.. use to use default 'stopCurrentJob=true'.. now pass false.

Very unlikely to make any difference, but in theory could timeout on semaphore then get a job.